### PR TITLE
cEP-0003: Add secret election policy

### PR DESCRIPTION
**Pull request by sims1253**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
ack bc2e66d6fc8fccda491bfc678d1b4fd4b4939f96

File: cEP-0003.md [hunk 1]
```diff
@@ -126,8 +126,9 @@ particular team forward. (Archived in the wiki.)
 The applications must be submit to the teams repository
 [here](https://github.com/coala/teams).
 
-Votings for Team Leaders represents a C Level Decision as specified in
-cEP-0001 and is done once a year.
+Elections for Team Leaders represents a C Level Decision as specified in
```
Comment (Adrianzatreanu) [inline cEP-0003.md]: represent*

**Comment by Adrianzatreanu** [comment]
unack bc2e66d6fc8fccda491bfc678d1b4fd4b4939f96

File: cEP-0003.md [hunk 2]
```diff
@@ -126,8 +126,9 @@ particular team forward. (Archived in the wiki.)
...
+cEP-0001 and is done once a year. In addition to the rules specified in
```
Comment (Adrianzatreanu) [inline cEP-0003.md]: are done

File: cEP-0003.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ Teams in coala
 |Metadata|                                                             |
 |--------|-------------------------------------------------------------|
 |cEP     |0003                                                         |
-|Version |1.2                                                          |
+|Version |1.2.1                                                          |
 |Title   |Teams in coala                                               |
```
Comment (Adrianzatreanu) [inline cEP-0003.md:8]: commit message: sais, really?

Comment (sims1253) [reply cEP-0003.md:8]: I don't understand this

Comment (aptrishu) [reply cEP-0003.md:8]: `cEP-0001 sais that a c level decision needs 1 week voting time` sais -> says

Comment (sims1253) [reply cEP-0003.md:8]: oh... my lord... I should not do this after drinking >.>

**Comment by aptrishu** [comment]
ack d7f763f

**Comment by aptrishu** [comment]
ack bc2e66d

**Comment by aptrishu** [comment]
reack bc2e66d

**Comment by sims1253** [comment]
@rultor merge

**Comment by aptrishu** [comment]
@rultor merge

**Comment by sims1253** [comment]
technical this pr is missing an ack but the decision to make voting public was discussed on in the gitter channel and was agreed on by at least another maintainer.

https://gitter.im/coala/coala/maintainers?at=583e96d98d65e3830ebacdb7

**Comment by sils** [comment]
second ack

**Pull request merged**